app/views/home/show.html.erb

```diff
@@ -0,0 +1,36 @@
+</br>
+<div class="button fa fa-arrow-left fa-lg pull-right">
+  <%= link_to 'Back', root_path, class: "btn btn-outline-dark" %>
+</div>
+
+<h2>Place <%= @place.id %></h2>
+
+  <ul>
+    <li><u><b>Name:</b></u> ........................................ <%= @place.name %></li>
```

Comment:
If you want you can add `list-group-flush` class. https://getbootstrap.com/docs/5.1/components/list-group/#flush

Reply:
Added here https://github.com/vova/war_shelter/pull/79/commits/7d0d1c05e2236d5b1982e5cee775fef00884219b

Reply:
Thanks!